docs/back-end-e2e.md

```diff
@@ -78,6 +78,18 @@ This document describes how to use E2E test locally.
 
 1. [install docker-compose](https://docs.docker.com/compose/install/)
 
+**NOTE:** To run docker compose locally, you need to modify ./api/conf/conf.yaml 'host' and 'endpoints' configuration in the file
```

Comment:
@Jaycean  How about here we say:  ( Thanks ) 


**NOTE:** In order to run docker compose locally, please change the values of `listen.host` and `etcd.endpoints` within `./api/conf/conf.yaml` as follows:

Reply:
Thks,Your translation is better than mine. Done.

Reply:
> Thks, It‘s better than mine. Done.

@Jaycean thanks.